package.json

```diff
@@ -79,5 +86,8 @@
   },
   "dependencies": {
     "dotenv": "^8.2.0"
+  },
+  "jest": {
```

Comment:
please investigate if we can remove this line here

Reply:
useless configuration removed